lib/active_admin/views/attributes_table.rb

```diff
@@ -4,7 +4,7 @@ module ActiveAdmin
   module Views
     class AttributesTable
       def bool_row(attribute, *args)
-        row(attribute, *args) { |model| model.send(attribute) ? status_tag('yes', :ok) : status_tag('no') }
+        row(attribute, *args) { |model| model.send(attribute) ? status_tag('yes', class: :ok) : status_tag('no') }
```

Comment:
@Ivanov-Anton  seems bool_row is not used anymore, better to delete this method

Reply:
Seems whole class can be removed


[link](https://github.com/activeadmin/activeadmin/commit/3efb359533df46e76bb4355a02ad31e727fb0da3#diff-0700093aaea7d2d82f67221074c1e94fae30a568354fa56f3d9e46cf9c04e60c)

Reply:
Removed in separate [PR](https://github.com/yeti-switch/yeti-web/pull/912)